<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoneyExchange</class>
 <widget class="QWidget" name="MoneyExchange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoneyExchange</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>309</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Account number:  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="accNum"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Amount of money:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="amountOfMoney">
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>PLN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EUR</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MoneyExchange</receiver>
   <slot>goBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MoneyExchange</receiver>
   <slot>sendMoney()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goBack()</slot>
  <slot>sendMoney()</slot>
  <slot>changeType()</slot>
 </slots>
</ui>
